pallets/rmrk-core/src/tests.rs

```diff
@@ -986,6 +995,79 @@ fn create_resource_works() {
 	});
 }
 
+/// Minting with resources works
+#[test]
+fn add_resource_on_mint_works() {
+	ExtBuilder::default().build().execute_with(|| {
+		let basic_resource: BasicResource<BoundedVec<u8, UniquesStringLimit>> =
+			BasicResource { src: None, metadata: None, license: None, thumb: None };
+
+		// Create a basic collection
+		assert_ok!(basic_collection());
+
+		let basic_resource =
+			BasicResource { src: None, metadata: None, license: None, thumb: None };
```

Comment:
This is more related to `add_resource` in general and rmrk spec but seems odd adding empty resources.